mkdocs.yml

```diff
@@ -1,5 +1,8 @@
 site_name: Brain Imaging Data Structure v1.2.0-dev
 theme: material
+markdown_extensions:
+    - toc:
+        permalink: "#"
```

Comment:
```suggestion
        anchorlink: true
```

Reply:
Great, if that does the trick I all all for it! Thanks!